docs/release/trg-0/trg-7-1.md

```diff
@@ -9,12 +9,103 @@ Proposed release date: "mandatory after": 19th of May 2023
 
 | Status | Created     | Post-History  |
 |--------|-------------|---------------|
-| Draft  | 24-Feb-2022 | n/a           |
+| Draft  | 13-Apr-2023 | Moved from OSS Development           |
 
 ## Why
 
-Describe in this section why the .
+Eclipse Tractus-X is an open source project hosted by the Eclipse Foundation licensed under the [Apache License 2.0](https://spdx.org/licenses/Apache-2.0). The legal obligations of the content must be observed in all forms of which the content is available.
+
+This page contains information about legal documentation requirements in your repositories. The reference is always the [Eclipse Foundation Project Handbook](https://www.eclipse.org/projects/handbook/#legaldoc).
+
+:::info
+
+The requirements described here must be met for each contribution.
+
+:::
 
 ## Description
 
-Describe the TRG on how.
+The following files have to be added at root level of your Eclipse Tractus-X repository:
+
+- LICENSE
+- NOTICE.md
+- DEPENDENCIES
+- SECURITY.md
+- CONTRIBUTING.md
+- CODE_OF_CONDUCT.md
+
+For examples look to the [Eclipse Tractus-X GitHub Organisation](https://github.com/eclipse-tractusx), e.g. the [APP Dashboard](https://github.com/eclipse-tractusx/app-dashboard).
+
+### LICENSE FILE
+
+In Eclipse Tractus-X the outbound license is Apache-2.0 for all code.
+
+- SPDX-License-Identifier: Apache-2.0
+- [License Text](https://www.apache.org/licenses/LICENSE-2.0.txt)
+
+See the [Handbook](https://www.eclipse.org/projects/handbook/#legaldoc-license).
+
+### NOTICE FILE
+
+- Add the link(s) to your repositories
```

Comment:
'to your repositories' i don't get this one. do you mean something like "To all connected repositories" or "all repositories part of this repository"?